WNPRC_EHR/resources/queries/study/water_amount.query.xml

```diff
@@ -0,0 +1,25 @@
+<query xmlns="http://labkey.org/data/xml/query">
+    <metadata>
+        <tables xmlns="http://labkey.org/data/xml">
+            <table tableName="Water Amount" tableDbType="TABLE">
```

Comment:
@dnicolalde is this dataset different than waterAmount?

Reply:
this an old version, I should remove it.